### ISSUE TITLE
px-deploy kubeconfig is returning private IP addresses for server (async-dr template)

### ISSUE THREAD

**Issue by bplein** [PureStorage-OpenConnect/px-deploy#204]
Tried async-dr template for the first time in a while (I am using px-deploy v 4.11) and using `px-deploy kubeconfig` brings back 2 kubeconfig files that refer to the AWS private IPs of the masters: `server: https://192.168.102.90:6443`

This is on AWS.

**Comment by bplein** [comment]
Oh, only tested with EKS. So marking this as an enhancement request for AWS.

**Comment by bplein** [comment]
this would be great to get working, so I can use kubectl from my laptop vs. `px-deploy connect`

**Comment by andrewh1978** [comment]
I agree. The problem is SSL certificates are tied to IPs and the internal IP is not the one you want to connect to. Any ideas?

**Comment by bplein** [comment]
yeah, I'm seeming to recall this... I guess it would be an AWS specific workaround (awscli?) to get the public IP of the master, and mark up the kubeconfig accordingly.  But no, that doesn't work because the API server is only on the private IP, not listening on the public. Not sure if it's a good idea anyhow.
